src/libs/actions/Receipt.ts

```diff
@@ -3,12 +3,8 @@ import ONYXKEYS from '../../ONYXKEYS';
 
 /**
  * Sets the upload receipt error modal content when an invalid receipt is uploaded
- *
- * @param {Boolean} isAttachmentInvalid
- * @param {String} attachmentInvalidReasonTitle
- * @param {String} attachmentInvalidReason
  */
-function setUploadReceiptError(isAttachmentInvalid, attachmentInvalidReasonTitle, attachmentInvalidReason) {
+function setUploadReceiptError(isAttachmentInvalid: boolean, attachmentInvalidReasonTitle: string, attachmentInvalidReason: string) {
```

Comment:
Don't we need to specify the return type? void in this case (same for the other function)

Reply:
According to [guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/TS_STYLE.md#:~:text=For%20function%20return%20types%2C%20default%20to%20always%20typing%20them%20unless%20a%20function%20is%20simple%20enough%20to%20reason%20about%20its%20return%20type.), I think this function is simple enough that we don't need to specify return type